frontend/public/components/utils/resource-log.jsx

```diff
@@ -26,8 +27,26 @@ const streamStatusMessages = {
   [STREAM_ACTIVE]: 'Log streaming...'
 };
 
+const isFullscreen = function() {
+  if (document.fullscreenElement) {
+    return true;
+  }
+  else if (document.webkitFullscreenElement) {
+    return true;
+  }
+  else if (document.mozFullScreenElement) {
+    return true;
+  }
+  else return false;
+};
+
 // Component for log stream controls
-const LogControls = ({dropdown, onDownload, status, toggleStreaming}) => {
+const LogControls = ({dropdown, onDownload, toggleFullscreen, status, toggleStreaming}) => {
+  const expandCompressClass = classNames({
```

Comment:
Since `fa` will always be used, this can be: 
```js
classNames( 'fa', {
    'fa-expand': !isFullscreen(), 
    'fa-compress': isFullscreen()
});
```